official/benchmark/benchmark_uploader_test.py

```diff
@@ -102,6 +102,22 @@ def test_upload_metric_file(self):
     self.mock_client.insert_rows_json.assert_called_once_with(
         self.mock_table, expected_params)
 
+  def test_insert_run_status(self):
+    self.benchmark_uploader.insert_run_status(
+        'dataset', 'table', 'run_id', 'status')
+    expected_query = ('INSERT dataset.table '
+                      '(run_id, status) '
+                      'VALUES(\'run_id\', \'status\')')
+    self.mock_client.query.assert_called_once_with(query=expected_query)
+
+  def test_update_run_status(self):
+    self.benchmark_uploader.update_run_status(
+        'dataset', 'table', 'run_id', 'status')
+    expected_query = ('UPDATE dataset.table '
+                      'SET status = \'status\' '
+                      'WHERE run_id = \'run_id\'')
+    self.mock_client.query.assert_called_once_with(query=expected_query)
```

Comment:
nit:  I think glint allows you to use double quotes when single quotes require escaping.